### PR TITLE
csvRead functional

### PR DESCRIPTION
**Pull request by chrnigi**
readData() can now read the dataset.

### CODE REVIEW

File: PowerHouse/CSV read/csvRead.c [hunk 1]
```diff
@@ -1,79 +1,103 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <time.h>
 #include "csvRead.h"
 
-Datapoint* readCSV(char *file_name)
+Datapoint* readData(char *filename)
```
Comment (theis999) [inline PowerHouse/CSV read/csvRead.c]: Change back please

File: PowerHouse/CSV read/csvRead.c [hunk 2]
```diff
@@ -1,79 +1,103 @@
...
-        printf("Kunne ikke åbne fil!\n");
-        return NULL;
-    }
+    // Uninitialized buffer to input the rows to.
+    char buffer[1024];
     int row = 0;
     int column;
-    while (fgets(buffer, 1024, fh))
+    // Allocation of the Datapoint-array, with the expected amounts of rows as the number of elements.
+    Datapoint *data = malloc(8760*sizeof(Datapoint));
```
Comment (theis999) [inline PowerHouse/CSV read/csvRead.c]: Magic numbers should be defined

**Review by theis999 — APPROVED** [review]
Looks fine outside of the name change

File: PowerHouse/CSV read/csvRead.c (hunk 2 above)
Comment (chrnigi) [reply PowerHouse/CSV read/csvRead.c]: Done, for now in .c file.

File: PowerHouse/CSV read/csvRead.c (hunk 1 above)
Comment (chrnigi) [reply PowerHouse/CSV read/csvRead.c]: Done

**Pull request merged**